scripts/run-ipv6-canary-test.sh

```diff
@@ -15,7 +15,7 @@ source "$SCRIPT_DIR"/lib/canary.sh
 function run_ginkgo_test() {
```

Comment:
Do we need to run the canary tests against the default addon version for ipv6? 

Also, can we fix the comment at the top of the file?

Reply:
I think we should run the IPv6 canary tests against the latest addon only. We can rely on the IPv4 canary script to provide the desired coverage for upgrading addons and basic functionality, while the IPv4 full script covers all IPv4 functionality against latest addon.

I also think that since the IPv6 canary tests do not do addon upgrade/downgrade, it makes sense to remove `install_add_on` and leave the responsibility of installing the latest addon to the cluster create step, which our prow already handles. That should actually speed IPv6 canary tests up a bit.

Reply:
Yes, this looks good. Thanks.